src/software/vasp.jl

```diff
@@ -354,3 +354,39 @@ end
 readPROCAR(filename::AbstractString) = open(readPROCAR, filename)
 
 readPROCAR() = open(readPROCAR, "PROCAR")
+
+"""
+    getFermi(io::IO) -> Float64
+
+Reads an OUTCAR file and returns the Fermi Energy.
+"""
+function getFermi(io::IO)
+    readuntil(io, "E-fermi :")
+    fermi = parse.(Float64, split(readline(io))[1])
+    return fermi
+end
+
+getFermi(filename::AbstractString) = open(getFermi, filename)
+
+getFermi() = open(getFermi, "OUTCAR")
```

Comment:
I recommend using snake case for this (so `get_fermi()` rather than `getFermi()`) to comply with the naming scheme used in other functions throughout Xtal.jl.